Description: revert acpi table size increase
 This is needed to let qemu -kernel work for now, until we figure out what
 else we have to change when the table size is increased
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: no
Index: qemu/hw/i386/acpi-build.c
===================================================================
--- qemu.orig/hw/i386/acpi-build.c
+++ qemu/hw/i386/acpi-build.c
@@ -62,7 +62,7 @@
 #define ACPI_BUILD_LEGACY_CPU_AML_SIZE    97
 #define ACPI_BUILD_ALIGN_SIZE             0x1000
 
-#define ACPI_BUILD_TABLE_SIZE             0x20000
+#define ACPI_BUILD_TABLE_SIZE             0x10000
 
 typedef struct AcpiCpuInfo {
     DECLARE_BITMAP(found_cpus, ACPI_CPU_HOTPLUG_ID_LIMIT);
